vcf-dumper/vcf-dumper-lib/src/test/java/embl/ebi/variation/eva/vcfdump/VariantExporterControllerTest.java

```diff
@@ -146,37 +141,37 @@ public void testGetQuery() throws Exception {
 
     @Test
     public void testVcfHtsExportOneStudy() throws ClassNotFoundException, URISyntaxException, InstantiationException, IllegalAccessException, IOException, IllegalOpenCGACredentialsException {
-        String studyId = "PRJEB6119";
+        String studyId = "7";
         List<String> studies = Collections.singletonList(studyId);
 
-        VariantExporterController controller = new VariantExporterController("btaurus", VariantExporterTestDB.COW_TEST_DB_NAME, studies, null, OUTPUT_DIR, evaTestProperties, emptyFilter);
+        VariantExporterController controller = new VariantExporterController("hsapiens", VariantExporterTestDB.TEST_DB_NAME, studies, null, OUTPUT_DIR, evaTestProperties, emptyFilter);
         controller.run();
 
         ////////// checks
         String outputFile = controller.getOuputFilePath();
         testOutputFiles.add(outputFile);
         assertEquals(0, controller.getFailedVariants());   // test file should not have failed variants
         QueryOptions query = getQuery(studies);
-        VariantDBIterator iterator = cowVariantDBAdaptor.iterator(query);
+        VariantDBIterator iterator = variantDBAdaptor.iterator(query);
         assertEqualLinesFilesAndDB(outputFile, iterator);
         checkOrderInOutputFile(outputFile);
     }
 
     @Test
     public void testVcfHtsExportSeveralStudies() throws Exception {
-        String study6119 = "PRJEB6119";
-        String study7061 = "PRJEB7061";
+        String study6119 = "7";
+        String study7061 = "8";
```

Comment:
These variable names should be renamed made less confusing, given they are no longer studies 6119 nor 7061.